cmd/init.go

```diff
@@ -18,9 +18,22 @@ func NewInitCommand(f Factory, ioStreams ioes.IOStreams) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "init [PATH]",
 		Short: "initialize a dataset directory",
-		Long: `'initialize' creates a new dataset, links it to the current or specified
-directory, and creates starter files for the dataset's components.`,
-		Example: ``,
+		Long: `'initialize' creates a new dataset, links it to the current directory,
+and creates starter files for the dataset's components. You can also specify an
+already existing body file using the 'body' flag.`,
+		Example: `  # initialize a new dataset, linking it to the current directory:
+
+  $ mkdir earthquakes && cd earthquakes
+  $ qri init
+  Name of new dataset [earthquakes]: 
+  Format of dataset, csv or json [csv]: csv
+  initialized working directory for new dataset ramfox/earthquakes
```

Comment:
Would suggest using a placeholder username here instead. A few lines down as well.

Reply:
do you mean like

`[USERNAME]/earthquakes`

or `my_username/earthquakes`

Reply:
Either, though I guess I have a preference for the second option. I don't think the docs have an explicit "example username" but maybe we should decide on one some point soon.

Reply:
after looking a bit, we use `b5`, `ramfox`, `chriswhong`, in the `qri log` examples. We also use `user` in `qri preview` and `qri remove` examples, so I'm going to go with `user`!